metrics/interface/src/test/resources/iotdb-metric.yml

```diff
@@ -24,6 +24,7 @@ enableMetric: true
 metricReporterList:
   - jmx
   - prometheus
+  - iotdb
```

Comment:
maybe iotdb is not a default value?

Reply:
I think users will only need one of  the two reporter : prometheus and iotdb.
So, it's better to let users choose what they really want.

Reply:
yes, so I think we don't need to set both as default value, maybe write in comment can be a good choice. As for what to use, it depends on users.

Reply:
Yes, I annotate iotdb and select prometheus as the default value.